### PR TITLE
fixing device id

### PR DESCRIPTION
**Pull request by dcritch**
I noticed when I 'fixed' the Intel IDs PR #16 , I typo'ed the proper ID, so I'm not getting results anymore.

Sorry!

### CODE REVIEW

**Comment by e-minguez** [comment]
According to the official documentation https://docs.openshift.com/container-platform/4.6/networking/hardware_networks/about-sriov.html#supported-devices_about-sriov the b is the supported one:

```
Intel XXV710 25GbE SFP28 with vendor ID 0x8086 and device ID 0x158b
```

**Comment by dcritch** [comment]
Thats why I got confused. On my cluster:
```
3b:00.0 Ethernet controller [0200]: Intel Corporation Ethernet Controller XXV710 for 25GbE backplane [8086:158a] (rev 02)
3b:00.1 Ethernet controller [0200]: Intel Corporation Ethernet Controller XXV710 for 25GbE backplane [8086:158a] (rev 02)
86:00.0 Ethernet controller [0200]: Intel Corporation Ethernet Controller XXV710 for 25GbE backplane [8086:158a] (rev 02)
86:00.1 Ethernet controller [0200]: Intel Corporation Ethernet Controller XXV710 for 25GbE backplane [8086:158a] (rev 02)
```

File: info/intel-firmware-version [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 # lspci -nn shows PCI vendor and device codes (and names)
-# Intel XXV710 25GbE SFP28 = 8086:158b
-IDS="8086:158b"
+# Intel XXV710 25GbE SFP28 = 8086:158a
+IDS="8086:158a"
```
Comment (e-minguez) [inline info/intel-firmware-version:5]: What about setting a default parameter that can be override? 
```suggestion
IDS="${IDS:=8086:158a}"
```

So you can:
```bash
export IDS="8086:158a"
openshift-checks -s info/intel-firmware-version
```

Didn't tested!

**Review by e-minguez — CHANGES_REQUESTED** [review]

**Comment by dcritch** [comment]
Made some changes

**Pull request merged**